src/app/containers/app/app.component.spec.ts

```diff
@@ -1,16 +1,20 @@
 import { TestBed, async } from '@angular/core/testing';
 import { AppComponent } from './app.component';
-import { LoggerService } from './logger/logger.service';
-
+import { LoggerService } from '../../service/logger/logger.service';
+import { RouterModule } from '@angular/router';
```

Comment:
https://angular.io/api/router/testing/RouterTestingModule probably best to use the router testing module within tests

Reply:
done